direct/tpm2/policy_test.go

```diff
@@ -281,6 +282,139 @@ func TestPolicyOrUpdate(t *testing.T) {
 	}
 }
 
+func getExpectedPCRDigest(t *testing.T, thetpm transport.TPM, selection tpml.PCRSelection, hashAlg tpm.AlgID) []byte {
+	t.Helper()
+	pcrRead := PCRRead{
+		PCRSelectionIn: selection,
+	}
+
+	pcrReadRsp, err := pcrRead.Execute(thetpm)
+	if err != nil {
+		t.Fatalf("Failed to read PCRSelection")
```

Comment:
(sorry to miss this before) nit: this should be something like "failed to read PCRs"

Reply:
Resolved.